gpu/ascend_linux.go

```diff
@@ -0,0 +1,101 @@
+//go:build linux
+
+package gpu
+
+/*
+#cgo linux LDFLAGS: -lrt -lpthread -ldl -lstdc++ -lm
+#cgo windows LDFLAGS: -lpthread
+
+#include "gpu_info_ascend.h"
+
+*/
+import "C"
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"path/filepath"
+	"strings"
+	"unsafe"
+)
+
+var AscendLinuxGlobs = []string{
+	"/usr/local/Ascend/latest/aarch64-linux/lib64/libascendcl.so*",
```

Comment:
Still can't find libascendcl.so in this path.

it's better to use: $ASCEND_TOOLKIT_HOME/lib64